features/bevragen/overzicht.md

```diff
@@ -59,6 +59,27 @@ De volgende velden hoeven niet met de fields parameter te worden gevraagd. Deze
 - opschortingBijhouding
 - rni
 
+### Filteren van verblijfplaats velden en de adresregel velden van de adressering gegevensgroep
+
+Voor verblijfplaats en adressering zijn er twee autorisatie profielen:
+
+- geautoriseerd voor verblijfplaats binnenland en verblijfplaats buitenland gegevens
+- geautoriseerd voor alleen verblijfplaats binnenland gegevens
+
+Consumers die geautoriseerd zijn voor alleen verblijfplaats binnenland gegevens kunnen hierdoor de standaard veld paden van verblijfplaats en adresregels niet gebruiken om alleen verblijfplaats binnenland velden te vragen. Met deze veld paden worden zowel verblijfplaats binnenland als verblijfplaats buitenland gevraagd.
+Om het mogelijk te maken voor consumers die niet geautoriseerd zijn voor het bevragen van verblijfplaats buitenland gegevens, zijn de volgende twee fields aliassen gedefinieerd waarmee kan worden aangegeven dat alleen verblijfplaats binnenland gegevens wordt opgevraagd: 
+
```

Comment:
Moeten we hier niet ook ergens verwijzen naar een van de csv bestanden?

Reply:
De vraag is wat de consumer met de csv bestanden moet. Persoonlijk zou ik elke verkorte pad met bijbehorende volledige pad opnemen in een scenario zodat hij direct kan zien welke paden hij kan gebruiken om een veld te bevragen